app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -212,6 +213,9 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
         requireWebRenderComponents.sessionUseCases.requestDesktopSite.invoke(enabled, session)
     }
 
+    override fun onContentPermissionRequested(session: Session, permissionRequest: PermissionRequest): Boolean =
```

Comment:
Is this for DRM?

Reply:
Yes, a-c updated the API for allowing permission requests to play DRM videos.